.travis.yml

```diff
@@ -77,10 +80,14 @@ matrix:
            CONDA_DEPENDENCIES=${CONDA_ALL_DEPENDENCIES}
            EVENT_TYPE='cron'
 
+
 install:
+  - sudo apt-get install libssl1.0.0
   - git clone git://github.com/astropy/ci-helpers.git
-  - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+  # - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
   # additional external tools (Issue #898) -- HOLE
+  - sudo pip install --upgrade pip
```

Comment:
Why do you need `sudo` and why do we need to upgrade `pip`? Isn't this taken care of by the conda installation? 

Avoiding use of `sudo` is desirable so that we can run on the Travis container-based infrastructure.